docs/website/docs/getting-started/tflite-cmd.md

```diff
@@ -0,0 +1,33 @@
+# TFLite via Command Line
```

Comment:
This page is small enough that it might make sense to collapse `tflite-cmd.md`, `tflite-python.md`, and `tflite.md` into a single page. Too many top level pages will bloat the navigation sidebar .

you can also use other features from mkdocs [material] to organize the page as needed:
https://squidfunk.github.io/mkdocs-material/reference/content-tabs/
https://squidfunk.github.io/mkdocs-material/reference/admonitions/

Reply:
Done.